changelogs/DP-22215.yml

```
@@ -0,0 +1,22 @@
+
+# Every PR must have a changelog. To create a changelog:
+# 1. Make a copy of this file and name it with the ticket number.
+# 2. Keep the original format of one of the examples at the bottom, and override each element of it using ___TEMPLATE___ for reference
+# 3. Remove all comments from the copied file
+
+#___CHANGE TYPES___
+# - Added for new features.
+# - Changed for changes in existing functionality.
+# - Deprecated for soon-to-be removed features.
+# - Removed for now removed features.
+# - Fixed for any bug fixes.
+# - Security in case of vulnerabilities.
+# e.g. Added
+
+#___EXAMPLE WITH MULTIPLE ITEMS___
+Changed:
+  - project: Patternlab
+    component: LocationListing
+    description: Replace googlemaps with leaftlet map, remove filter logic.
+    issue: DP-22215
+    impact: Major
```

```suggestion
    impact: Minor 
```
Let's change this to minor, as we have been releasing these as minor versions before